echopype/echodata/echodata.py

```diff
@@ -403,12 +403,29 @@ def update_platform(
             extra_platform_data = extra_platform_data.drop_vars(trajectory_var)
             extra_platform_data = extra_platform_data.swap_dims({"obs": time_dim})
 
-        # TODO: Expand the clipping range by one (or 2?) indices
+        # clip incoming time to 1 less than min of EchoData.beam["ping_time"] and
+        #   1 greater than max of EchoData.beam["ping_time"]
+        # account for unsorted time
```

Comment:
> `# account for unsorted time`

Which unsorted time are you referring to? is it EchoData.beam."ping_time" or the time in the external platform location time? From code the below it seems to be both? Would be great to add that to improve code readability.